File: PHP/Cap07/c07ex06.php
<?php
// c07ex06.php

function FATORIAL($N): int {
  $Fat = 1;
  for ($I = 1; $I <= $N; $I++) {
    $Fat *= $I;
  }
  return $Fat;
}

$Limite;

printf("Qual fatorial: ");
$Limite = intval(trim(fgets(STDOUT)));

printf("Fatorial = %d\n", FATORIAL($Limite));

printf("\n");
printf("Tecle <Enter> para encerrar... ");
trim(fgets(STDOUT));

?>

File: VB_NET/Cap07/c07ex06.vb
' c07ex06.vb

Imports System
Imports System.Console

Module Program
    Function Fatorial(ByVal N As Integer) As Integer
        Dim Fat As Integer = 1
        For I As Integer = 1 To N
            Fat *= I
        Next
        Return Fat
    End Function

    Sub Main(args As String())
        Dim Limite As Integer

        Write("Qual fatorial: ")
        Limite = Integer.Parse(ReadLine())

        WriteLine($"Fatorial = {Fatorial(Limite)}")

        Write(vbCrLf & "Tecle <Enter> para encerrar... ")
        ReadLine()
    End Sub
End Module
